approvals/approvals/doctype/document_approval_rule/document_approval_rule.py

```diff
@@ -103,12 +106,11 @@ def assign_user(self, doc: Document):
 			todo.status = "Open"
 			todo.priority = "Medium"
 			todo.document_approval_rule = self.name
+			todo.rejection = rejection
 			todo.description = (
 				self.get_message(doc) if self.message else frappe._("A document has been assigned to you")
```

Comment:
I left the `message` field because is used here for the `ToDo` description but we can delete it and leave the generic message.